packages/bridge-ui/src/components/form/BridgeForm.svelte

```diff
@@ -215,6 +216,24 @@
       if (requiresAllowance) throw Error("requires additional allowance");
 
       const amountInWei = ethers.utils.parseUnits(amount, $token.decimals);
+
+      let isBridgedTokenAlreadyDeployed = false;
```

Comment:
@cyberhorsey What do you think about this way of checking if an increased `gasLimit` is needed?